app/controllers/admin/snap_applications_controller.rb

```diff
@@ -17,5 +17,25 @@ class SnapApplicationsController < Admin::ApplicationController
 
     # See https://administrate-prototype.herokuapp.com/customizing_controller_actions
     # for more information
+    #
+    def resend_fax
+      application = SnapApplication.find(params[:id])
+      application.exports.faxed.succeeded.update_all(
+        status: :failed,
+        metadata: "Reset by admin",
+      )
+      Export.create_and_enqueue!(snap_application: application, destination:
+                                 :fax)
+      flash[:notice] = "Resent fax #{application.email}!"
```

Comment:
this flash message is kind of confusing -- we faxed an email? Also, email could be nil so this flash message might be unhelpful in some cases